postfix.grok

```diff
@@ -16,6 +16,7 @@ POSTFIX_TLSCONN (Anonymous|Trusted|Untrusted|Verified) TLS connection establishe
 POSTFIX_DELAYS %{NUMBER:postfix_delay_before_qmgr}/%{NUMBER:postfix_delay_in_qmgr}/%{NUMBER:postfix_delay_conn_setup}/%{NUMBER:postfix_delay_transmission}
 POSTFIX_LOSTCONN (lost connection|timeout|Connection timed out)
 POSTFIX_PROXY_MESSAGE (%{POSTFIX_STATUS_CODE:postfix_proxy_status_code} )?(%{POSTFIX_STATUS_CODE_ENHANCED:postfix_proxy_status_code_enhanced})?.*
+POSTFIX_CONNSTATUS %{POSTFIX_QUEUEID:postfix_queueid}: host %{POSTFIX_RELAY_INFO} said: %{POSTFIX_STATUS_CODE:postfix_status_code} %{POSTFIX_STATUS_CODE_ENHANCED:postfix_status_code_enhanced} <%{GREEDYDATA:postfix_sender_address}>: %{GREEDYDATA:postfix_connect_status}
```

Comment:
This project is about parsing postfix logging. I have no intention to add support for a subset of logging from other MTAs, since the variants will be too wide, and there are too many MTAs to support.

Adding additional patterns to logstash to process the MTA responses should be quite simple, since then you can simply support only the messages useful to you. Logstash is clearly designed for this kind of approach.